tests/unit/test_core.py

```diff
@@ -0,0 +1,85 @@
+from openscm.core import (
+    ParameterAggregatedError,
+    ParameterReadonlyError,
+    ParameterSet,
+    ParameterType,
+    ParameterTypeError,
+    ParameterWrittenError,
+    RegionAggregatedError,
+)
+import pytest
+
+parameterset = ParameterSet()
```

Comment:
turn this into a fixture so it's regenerated before each test

Reply:
What do you mean?

Reply:
parameterset should be a pytest fixture so we get a fresh instance of ParameterSet for each test. The way you have it set up, the same instance of ParameterSet is used in all of the tests and hence we could get some weird fails because something that was set in one test blows things up in another (when it shouldn't).

Reply:
done